mne/externals/FieldTrip.py

```diff
@@ -309,7 +309,7 @@ def getHeader(self):
                 (chunk_type, chunk_len) = struct.unpack(
                     'II', payload[offset:offset + 8])
                 offset += 8
-                if offset + chunk_len < bufsize:
+                if offset + chunk_len > bufsize:

```

Comment:
That's cool. Now I don't have to use the FieldTrip file streamer anymore (had enough Matlab for a lifetime...)